### ISSUE TITLE
Rounding issues cause deactivated users to reappear indefinitely

### ISSUE THREAD

**Issue by pH142857** [spiral-project/ihatemoney#1336]
Users who have been set to a balance of zero and deactivated keep reappearing with a balance of ±0,01. I thought I could get rid of the problem by make up for it with a fake transaction, but I can't: bad rounding causes a specific user to have either +0,01 or -0,01. What's more, to create a fake transaction I have to reactivate the user, and this alone is a problem because of possible side effects.

It is hard to replicate but if you have access to serverside data I can tell you which project have this issue on the hosted service.

By the way, is this project maintained? It doesn't seem very active.

**Comment by almet** [comment]
> It is hard to replicate but if you have access to serverside data I can tell you which project have this issue on the hosted service.

Yep, that would be welcome, so we can have a look, thanks for opening an issue :-)

> By the way, is this project maintained? It doesn't seem very active.

You're right, we're doing the minimum service right now, and are in maintainance mode, see https://github.com/spiral-project/ihatemoney?tab=readme-ov-file#current-direction-as-of-2024 for more info.

**Comment by pH142857** [comment]
Great, I'm all for stability and simplicity!
Concern project is `coloc-du-77`. Concerned username is `Natan`.
